.github/workflows/deploy_cms.yaml

```
@@ -0,0 +1,90 @@
+name: Deploy CMS
+
+on:
+  push:
+        branches: 'main'
```

```suggestion
        branches: ['main']
```
`branches` need an array, so you can do as I suggested or even
```
branches:
    - branch1
    - branch2
```